Stripe/StripeiOS/Source/STPAPIClient+PushProvisioning.swift

```diff
@@ -37,4 +37,10 @@ extension STPAPIClient {
             completion(details, error)
         }
     }
+    // MARK: Helpers
+
+    /// A helper method that returns the Authorization header to use for API requests. If ephemeralKey is nil, uses self.publishableKey instead.
+    func authorizationHeader(using ephemeralKey: STPEphemeralKey? = nil) -> [String: String] {
```

Comment:
Was this moved from somewhere?

Reply:
Yes, this was from [STPAPIClient+BasicUI.swift](https://github.com/stripe/stripe-ios/blob/c8f86df3/Stripe/StripeiOS/Source/STPAPIClient+BasicUI.swift#L173), but I moved it into the STPAPIClient+PushProvisioning file because that's where the function was used and I didn't want there to be any leftover +BasicUI files